### ISSUE TITLE
Upgrade to Hystrix 1.5

### ISSUE THREAD

**Issue by rhart** [ratpack/ratpack#934]
See the 1.5.0 release notes for some of the differences https://github.com/Netflix/Hystrix/releases/tag/v1.5.0

And also https://github.com/Netflix/Hystrix/wiki/Metrics-and-Monitoring

**Comment by hyleung** [comment]
I tried bumping Hystrix to 1.5.1 and I'm seeing a test failure due to an exception in `HystrixRequestCachingSpec` (`can cache #executionType command execution`):

```
Caused by: java.lang.ClassCastException: ratpack.rx.internal.MultiExecControllerBackedScheduler cannot be cast to rx.internal.schedulers.EventLoopsScheduler
	at rx.internal.util.ScalarSynchronousObservable$1.<init>(ScalarSynchronousObservable.java:42)
	at rx.internal.util.ScalarSynchronousObservable.<clinit>(ScalarSynchronousObservable.java:41)
	... 100 more
```

Looks like in RxJava they're expecting the computation scheduler to always be an `EventLoopsScheduler`.

```
/**
 * We expect the Schedulers.computation() to return an EventLoopsScheduler all the time.
*/
static final Func1<Action0, Subscription> COMPUTATION_ONSCHEDULE = new Func1<Action0, Subscription>() {
    final EventLoopsScheduler els = (EventLoopsScheduler)Schedulers.computation();
        
    @Override
    public Subscription call(Action0 t) {
        return els.scheduleDirect(t);
    }
};
```

**Comment by danveloper** [comment]
This was fixed in #944
